Frontend/src/components/I18n.js

```diff
@@ -0,0 +1,135 @@
+
+        // Student Section
+        student_title: "L'aventure de dialogue ultime",
+        student_description: "Rejoignez cette aventure en ligne pour apprendre les compétences du dialogue, explorer différentes perspectives et sujets mondiaux, et établir des connexions interculturelles avec des jeunes du monde entier ! Grâce à ce jeu amusant et interactif, vous pourrez :",
+        student_feature1: "Recevez des badges pour avoir complété des formations sur différents sujets.",
+        student_feature2: "Suivez vos progrès grâce à des tableaux de bord personnalisés.",
+        student_feature3: "Collaborez avec vos pairs dans des salles de classe virtuelles.",
+        student_feature4: "Accédez à des ressources sélectionnées par des experts.",
+
+        // Teacher Section
+        teacher_title: "Accès aux ressources pour la classe",
+        teacher_description: "Explorez les ressources, outils et formations pour enseigner et développer les compétences et les capacités du dialogue dans votre communauté d'apprentissage. Inscrivez-vous ici pour faire partie du réseau Generation Global et recevoir un accès à :",
+        teacher_feature1: "Ressources pédagogiques téléchargeables, y compris les Essentiels du Dialogue",
+        teacher_feature2: "Invitez vos élèves et suivez leur parcours à travers l'Aventure Ultime du Dialogue",
+        teacher_feature3: "Réservez des vidéoconférences pour que vos élèves puissent dialoguer avec des pairs mondiaux",
+        teacher_feature4: "Opportunités de développement professionnel",
+        
+      },
+    },
+    es: {
+      translation: {
+        welcome: "Bienvenido",
+        description: "Esta es una aplicación internacionalizada.",
+        about_us: "Sobre nosotros",
+        our_work: "Nuestro trabajo",
+        news: "Noticias",
+        resources: "Recursos",
+        events: "Eventos",
+        login_join_free: "Iniciar sesión o Registrarse gratis",
+        for_students: "Para estudiantes",
+        for_teachers: "Para profesores",
+        start_adventure: "Comienza la aventura ahora",
+        login: "Iniciar sesión",
+        student_companion: "Guía para estudiantes",
+        teacher_companion: "Guía para profesores",
+
+        // Student Section
+        student_title: "La Aventura del Diálogo Definitiva",
+        student_description: "¡Únete a esta aventura en línea para aprender las habilidades del diálogo, explorar diferentes perspectivas y temas globales, y construir conexiones interculturales con jóvenes de todo el mundo! A través de este juego divertido e interactivo, podrás:",
+        student_feature1: "Recibir insignias por completar el aprendizaje en diferentes temas",
+        student_feature2: "Ganar puntos en tu camino para convertirte en un maestro del diálogo",
+        student_feature3: "Descubrir formas en las que puedes usar el diálogo en la vida real con compañeros globales",
+        student_feature4: "Practicar tus habilidades de diálogo",
+        
```

Comment:
What the fuck is this? 
Why the FUCK would you leave so much space here?
Don't you fucking check before you commit a change